clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -254,7 +254,7 @@ message Query {
 message SearchRequest {
     string indexName = 1; //index to search against
     int32 startHit = 2; // Which hit to start from (for pagination); default: 0
-    int32 topHits = 3; // How many top hits to retrieve; default: 10
+    int32 topHits = 3; // How many top hits to retrieve; default: 10. For pagination: set it to startHit + window_size. 
```

Comment:
Hmm, although I think its already broken since only the comments that go on a line before the field itself show up in swagger_UI
<img width="815" alt="Screen Shot 2021-03-05 at 10 14 01 AM" src="https://user-images.githubusercontent.com/4211071/110156201-85048180-7d9b-11eb-8133-f2fe03c6c76c.png">